sentry/release_deployment.go

```diff
@@ -0,0 +1,61 @@
+package sentry
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+type ReleaseDeploymentsService service
+
+type ReleaseDeployment struct {
+	ID           string     `json:"id"`
+	Name         *string    `json:"name,omitempty"`
+	Environment  string     `json:"environment,omitempty"`
+	URL          *string    `json:"url,omitempty"`
+	Projects     []string   `json:"projects,omitempty"`
+	DateStarted  *time.Time `json:"dateStarted,omitempty"`
+	DateFinished *time.Time `json:"dateFinished,omitempty"`
+}
+
+// Get a Release Deploy for a project.
+func (s *ReleaseDeploymentsService) Get(ctx context.Context, organizationSlug string, version string, deployID string) (*ReleaseDeployment, *Response, error) {
+	u := fmt.Sprintf("0/organizations/%v/releases/%s/deploys/", organizationSlug, version)
```

Comment:
Thanks for opening the PR!

It looks like this endpoint has pagination. [See the source code here](https://github.com/getsentry/sentry/blob/707b8b0b2d15b3380af4b8e571032f6628d8f5eb/src/sentry/api/endpoints/release_deploys.py#L55-L61).

Could you please paginate through the pages to find the deployment?

[The Response struct has a Cursor property](https://github.com/jianyuan/go-sentry/blob/9a37263e941fa308e5923dc165470cf9589d214c/sentry/sentry.go#L189-L198) you can use. [Here is an example usage](https://github.com/jianyuan/go-sentry/blob/9a37263e941fa308e5923dc165470cf9589d214c/sentry/organization_integrations.go#L44-L68).

Reply:
Updated the code to paginate through all pages